python/ray/rllib/agents/ddpg/ddpg_policy_graph.py

```diff
@@ -378,3 +467,17 @@ def get_state(self):
     def set_state(self, state):
         TFPolicyGraph.set_state(self, state[0])
         self.set_epsilon(state[1])
+
+    def build_compute_apply(self, builder, postprocessed_batch):
+        builder.add_feed_dict(self.extra_compute_grad_feed_dict())
+        builder.add_feed_dict(self.extra_apply_grad_feed_dict())
+        builder.add_feed_dict(self._get_loss_inputs_dict(postprocessed_batch))
+        builder.add_feed_dict({self._is_training: True})
+        fetches = builder.add_fetches([
+            self._apply_op if self.policy_delay_count %
+            self.config["policy_delay"] == 0 else self._apply_ops[0],
```

Comment:
Would it be more natural to multiply the gradient by 0 unless policy delay is mod 0? That way it is not necessary to add these hacks in the policy graph.

I am also kind of skeptical of this optimization, seems like you can probably just tune some loss coefficients instead.